Firmware/MotorControl/motor.cpp

```diff
@@ -440,11 +440,19 @@ bool Motor::FOC_current(float Id_des, float Iq_des, float I_phase, float pwm_pha
 }
 
 
-bool Motor::update(float current_setpoint, float phase, float phase_vel) {
-    current_setpoint *= config_.direction;
+bool Motor::update(float torque_setpoint, float phase, float phase_vel) {
+    float current_setpoint;
```

Comment:
Maybe set to 0 by default to avoid magic smoke in case of a code error in the future.